### PR TITLE
Added 'from' arg to send-mail command

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/17776

## Description
Added 'from' arg to send-mail command.

## Does it break backward compatibility?
- [x] No

### CODE REVIEW

**Review by yuvalbenshalom — COMMENTED** [review]
@ChanochShayner this looks good. please checkout the tests in the build.
also, I am not sure, maybe this can also be checked by the existing test playbook?

File: Packs/MicrosoftGraphListener/ReleaseNotes/1_0_22.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 Outlook Mail Single User (Using Graph API)
+- Added **from** argument to the ***send-mail*** command as email address from which to reply.
```
Comment (guykeller) [inline Packs/MicrosoftGraphListener/ReleaseNotes/1_0_22.md:4]: Please change all the RN accordingly
```suggestion
- Added the **from** argument to the ***send-mail*** command, which represents the email address from which to reply.
```

**Review by guykeller — APPROVED** [review]
Please change the RN to what I mentioned. Other than that, approved

**Comment by guykeller** [comment]
@ChanochShayner If possible, add a test to the existing playbook.
If not, only change the RN and merge

**Comment by ChanochShayner** [comment]
My branch is full of conflicts and stacking.

**Pull request closed without merging**